client/src/components/AdvisorReports/AdvisorReportsTableHead.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import Checkbox from 'components/Checkbox'
+
+const AdvisorReportsTableHead = (props) => {
+    let weekHeadings = []
```

Comment:
I would add here the propTypes. That way you force certain property types and you also document the component better.